app/views/views.py

```diff
@@ -153,15 +153,19 @@ def download_file(request: HttpRequest):
             except TimeoutError:
                 print(f"{task} {task_name} failed #{attempts}")
                 if (attempts >= ATTEMPT_LIMIT):
-                    return _error(request, "Download exceeded max attempts")
+                    return _error(
+                        request=request,
+                        err_msg="Download exceeded max attempts")
         return result
     except Exception as e:
-        return _error(request, "Something went wrong.", e)
+        return _error(request=request, err_msg=f"Failed to download file: {e}")
 
```

Comment:
Sanity check: I assume we don't need to parse the Exception to be string in Python unlike JS?

Reply:
Right, the f-string handles that automatically.